newIDE/app/package.json

```diff
@@ -92,7 +92,9 @@
     "analyze-source-map": "source-map-explorer build/static/js/main.*",
     "extract-all-translations": "node scripts/extract-all-translations.js",
     "compile-translations": "node scripts/compile-translations.js",
-    "reload-extensions": "node scripts/import-GDJS-Runtime.js"
+    "reload-extensions": "node scripts/import-GDJS-Runtime.js",
+    "import-zipped-external-editors": "cd scripts && node import-zipped-editor.js piskel 5.0.0-beta82 e02f67866b338b4cc577e1ffa48f55f52cd520ed751dccc6f0931e3c1b995de0 && node import-zipped-editor.js jfxr 5.0.0-beta55 8ac12b557c2ddba958c6f0d3e0c5df8cf3369a65262dcb90cf5c8a7a7d20bdf6 && node import-zipped-editor.js yarn 5.0.0-beta80 5930c686ccf2b6fd3433b1c9539483473a6aedb8b6a73dd13c07030c95efecb9",
+    "import-zipped-electron-extensions": "cd scripts && node import-zipped-file.js ReactDeveloperTools ../../electron-app/app/extensions/"
```

Comment:
I wonder if this should be in `electron-app/package.json` instead. This is Electron specific, so as possible we should avoid polluting the generic app sources with installation of extensions?